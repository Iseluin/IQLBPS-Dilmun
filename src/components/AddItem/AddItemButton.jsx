import ADD_ITEM_ROUTE from '/add-item';
function addItem() {
  return ( 
<div>
<div className="btn__container">
            <button type='button' className="bg-secondary hover:bg-tertiary text-white font-bold py-2 px-4 rounded-full">  
            Add Item
            </button>
          </div>
</div>
  )
}

export default addItem;