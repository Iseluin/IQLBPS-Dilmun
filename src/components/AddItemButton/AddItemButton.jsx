import { Link } from 'react-router-dom';

function AddItemButton() {
  return (
    <Link to="/add-item">
      <div className="btn__container">
        <button
          type="button"
          className="bg-secondary hover:bg-tertiary text-white font-bold py-2 px-4 rounded-lg shadow-md hover:shadow-none transition duration-200"
        >
          Add Item
        </button>
      </div>
    </Link>
  );
}
export default AddItemButton;
