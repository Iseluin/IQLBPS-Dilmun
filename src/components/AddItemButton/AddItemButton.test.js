import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import AddItemButton from "./AddItemButton";


test('Add item button component rendered correctly', () => {
    const component = renderer.create(<BrowserRouter><AddItemButton /></BrowserRouter>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})