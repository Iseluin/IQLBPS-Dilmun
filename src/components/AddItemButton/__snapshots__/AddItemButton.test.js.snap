// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add item button component rendered correctly 1`] = `
<a
  href="/add-item"
  onClick={[Function]}
>
  <div
    className="btn__container"
  >
    <button
      className="bg-secondary hover:bg-tertiary text-white font-bold py-2 px-4 rounded-lg shadow-md hover:shadow-none transition duration-200"
      type="button"
    >
      add_item_button.add_item
    </button>
  </div>
</a>
`;
